libipl/p10/ipl0.C

```diff
@@ -374,28 +387,137 @@ static int ipl_alignment_check(void)
 	return -1;
 }
 
+static int initialize_and_check_clock_chip(uint8_t& clock_count)
+{
+	struct pdbg_target *clock_target;
+	enum pdbg_target_status status;
+	uint8_t data;
+	int i2c_rc = 0;
+	int rc = 0;
+
+	// initialize output variable.
+	clock_count = 0;
+
+	ipl_log(IPL_DEBUG, "Istep: soft reset clock, and verify clock status register\n");
+	pdbg_for_each_class_target("oscrefclk", clock_target) {
+		if(!ipl_is_functional(clock_target)){
+			continue;
+		}
+
+		// clock count has to be incremented even if it failed to initiallize
+		clock_count++;
+
+		status = pdbg_target_probe(clock_target);
+		if(status != PDBG_TARGET_ENABLED){
```

Comment:
File scope: I think you need to add a `space` before the curly place (a few places you added but, missed in a few places).

Reply:
do you mean there should be a space, between line 410 & 411.

Reply:
So for the confusion (typo in my comments)

Need space at LN 411 after `if` and before `curly brace`
```
if(status != PDBG_TARGET_ENABLED){
```
to
```
if (status != PDBG_TARGET_ENABLED) {
```

Reply:
Ok